var/spack/repos/builtin/packages/gdbm/package.py

```diff
@@ -46,4 +46,12 @@ class Gdbm(AutotoolsPackage):
     depends_on("readline")
 
     def configure_args(self):
-        return ['--enable-libgdbm-compat']
+
+        # GDBM uses some non-standard GNU extensions,
+        # enabled with -D_GNU_SOURCE.  See:
+        #   https://patchwork.ozlabs.org/patch/771300/
+        #   https://stackoverflow.com/questions/5582211/what-does-define-gnu-source-imply?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa
```

Comment:
You don't need the full URL here, https://stackoverflow.com/questions/5582211 is sufficient.